modules/installation-mirror-repository.adoc

```diff
@@ -145,6 +145,10 @@ the `imageContentSources` data that you require when you install your cluster.
 
 ... Record the entire `imageContentSources` section from the output of the previous
 command. The information about your mirrors is unique to your mirrored repository, and you must add the `imageContentSources` section to the `install-config.yaml` file during installation.
+[NOTE]
```

Comment:
This is building a bit wonky. Try adding a blank line before the [NOTE].

Reply:
s/quay.io/Quay.io

Reply:
Your edits have been incorporated. Thank you!